bench/.gitignore

```diff
@@ -0,0 +1,3 @@
+benchmark.csv
```

Comment:
What are these lines for?

Reply:
Is my gitignore file, it can be ignored, I have deleted it for the next pull